From 8d7be9fcfdeb1a59c18408dbabc30842854631a4 Mon Sep 17 00:00:00 2001
From: Andrey Konovalov <andrey.konovalov@linaro.org>
Date: Fri, 18 Aug 2017 17:19:51 +0300
Subject: [PATCH] Dont'check for gbm version, do check egl version required by
 cube-video

gbm in meta-ti has the version number of 2.0.0 which doesn't match
the check in configure.ac.

cube-video code uses EGL version 15 features, and would not build
with an older EGL version.

Signed-off-by: Andrey Konovalov <andrey.konovalov@linaro.org>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8397f7b..7788d85 100644
--- a/configure.ac
+++ b/configure.ac
@@ -36,12 +36,12 @@ m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
 # Obtain compiler/linker options for depedencies
 PKG_CHECK_MODULES(DRM, [libdrm >= 2.4.71])
-PKG_CHECK_MODULES(GBM, gbm >= 13.0)
+PKG_CHECK_MODULES(GBM, gbm)
 PKG_CHECK_MODULES(EGL, egl)
 PKG_CHECK_MODULES(GLES2, glesv2)
 
 # Check for gst and enable cube-video conditionally:
-PKG_CHECK_MODULES(GST, gstreamer-1.0 >= 1.6.0 gstreamer-plugins-base-1.0 >= 1.6.0 gstreamer-app-1.0 >= 1.6.0 gstreamer-allocators-1.0 >= 1.6.0 gstreamer-video-1.0 >= 1.6.0 glib-2.0,
+PKG_CHECK_MODULES(GST, gstreamer-1.0 >= 1.6.0 gstreamer-plugins-base-1.0 >= 1.6.0 gstreamer-app-1.0 >= 1.6.0 gstreamer-allocators-1.0 >= 1.6.0 gstreamer-video-1.0 >= 1.6.0 glib-2.0 egl >= 15.0.0,
 		 [HAVE_GST=yes], [HAVE_GST=no])
 if test "x$HAVE_GST" = "xyes"; then
 	AC_DEFINE(HAVE_GST, 1, [Have GStreamer support])
-- 
2.7.4

